clients/fides-js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "fides-js",
+  "version": "0.0.1",
+  "description": "Fides Javascript",
+  "license": "Apache-2.0",
+  "main": "./dist/fides.global.js",
```

Comment:
TODO: this would be better as `fides.js`, the `.global.js` doesn't have the same ring to it